sys-fs/python-btrfs/python-btrfs-13.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+DISTUTILS_EXT=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python module to inspect btrfs filesystems"
+HOMEPAGE="https://github.com/knorrie/python-btrfs"
+SRC_URI="https://github.com/knorrie/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="LGPL-3"
+KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
You tested on arm64&x86?

Reply:
ooohh.. *that*'s what the ext means.. I thought it was a distutil extension :zany_face: 
There's no C code (guaranteed) and I've removed the DISTUTILS_EXT line.